data/human/ships.txt

```diff
@@ -2451,6 +2451,55 @@ ship "Hogshead"
 
 
 
+ship "Jackal"
+	sprite "ship/jackal"
+	thumbnail "thumbnail/jackal"
+	attributes
+		category "Interceptor"
+		"cost" 320000
+		"shields" 1600
+		"hull" 500
+		"required crew" 1
+		"bunks" 2
+		"mass" 100
+		"drag" .95
+		"heat dissipation" .68
+		"fuel capacity" 500
+		"cargo space" 10
+		"outfit space" 155
+		"weapon capacity" 25
+		"engine capacity" 40
+		weapon
+			"blast radius" 21
+			"shield damage" 210
+			"hull damage" 105
+			"hit force" 315
+	outfits
+		"Modified Blaster" 4
+		
+		"nGVF-BB Fuel Cell"
+		"LP036a Battery Pack"
+		"D14-RN Shield Generator"
+		
+		"Chipmunk Plasma Thruster"
+		"Chipmunk Plasma Steering"
+		"Hyperdrive"
+
+	engine -10 35
+	engine 10 35
+	gun -17 -8.5
+	gun 17 -8.5
+	gun -25 -6
+	gun 25 -6
+	leak "leak" 50 40
+	leak "flame" 70 80
+	explode "tiny explosion" 10
+	explode "small explosion" 10
+	"final explode" "final explosion small"
+	description "The Jackal is notable for being one of the earliest warships that was designed in mind for "freelancer" pilots, rather than for corporate or law enforcement usage. In its heyday, the Jackal contended with the Berserker for the title of the most popular Interceptor-class ship in human space. However, the Jackal's age and inflexibility with regards to new developments in weaponry led to a decline in sales that eventually bankrupted its manufacturer, ending a century-long production run - a phenomenon its age-old rival, the Berserker, may only have avoided thanks to Betelgeuse Shipyards' other, more popular models."
```

Comment:
```suggestion
	description `The Jackal is notable for being one of the earliest warships that was designed in mind for "freelancer" pilots, rather than for corporate or law enforcement usage. In its heyday, the Jackal contended with the Berserker for the title of the most popular Interceptor-class ship in human space. However, the Jackal's age and inflexibility with regards to new developments in weaponry led to a decline in sales that eventually bankrupted its manufacturer, ending a century-long production run - a phenomenon its age-old rival, the Berserker, may only have avoided thanks to Betelgeuse Shipyards' other, more popular models.`
```